### ISSUE TITLE
Failed to install v1.8.0

### ISSUE THREAD

**Issue by Wigithub1201** [heartexlabs/label-studio-frontend#1439]
Hi 

My system is Windows 10, and I executed the commands ‘npm install @heartexlabs/label studio’ and ‘yarn add @heartexlabs/label’ studio, but the dependent packages were not successfully installed. What is the reason @makseq 

![image](https://github.com/heartexlabs/label-studio-frontend/assets/27763586/45eb25b4-0bdf-4987-955e-8987efd5ae76)

**Comment by Wigithub1201** [comment]
There are only images and no build folder on the NPM package
![image](https://github.com/heartexlabs/label-studio-frontend/assets/27763586/38afeed4-5426-430f-9e09-518594476e39)

**Comment by lbelmonte-ag** [comment]
Same error here. We cannot install via npm the 1.8.0 version. 1.4.0 still works fine

**Comment by YukiYamamoto228** [comment]
I have the same error on Linux.
I tried to build from source, but could not find a way to properly export the 'LabelStudio' module.

**Comment by bigabig** [comment]
Hi have you found a solution? I am facing the same issue unfortunately

**Comment by Zensome** [comment]
Same problem here! hopefully the LSF team can resolve this! thanks alot!

**Comment by ruanhan** [comment]
npm install @heartexlabs/label-studio --ignore-scripts

**Comment by ruanhan** [comment]
https://github.com/heartexlabs/label-studio-frontend/issues/1488

**Comment by gigaenr-bh** [comment]
same problem here

**Comment by jheyming** [comment]
Any chance to fix this?

**Comment by Oguerisck** [comment]
Same error here

**Comment by AndrewLester** [comment]
Try installing v1.4.0

```bash
npm i @heartexlabs/label-studio@1.4.0
```

**Comment by tranhd95** [comment]
> Has anyone found a fix for this?
> 
> I am facing same error
> 

The package is broken (not uploaded correctly) in NPM repository. If you want to use it, I'd recommend pulling the repository and build the package not with npm but with yarn.

tl;dr try to replicate the steps from GitHub CI and pay attention to versions used.
https://github.com/HumanSignal/label-studio-frontend/blob/master/.github/workflows/build_bundle.yml

**Comment by markotonic** [comment]
Is there a workaround for this other than building the package locally and pointing to that? https://github.com/HumanSignal/label-studio-frontend/issues/1488 is also open with the same issue
